tests/python/unittest/test_tir_sparse_scripts.py

```diff
@@ -0,0 +1,96 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import tvm
+import tvm.tir as tir
+import tvm.te as te
+from tvm.script import tir as T
+
+
+@T.prim_func
+def csrmm(a: T.handle, b: T.handle, c: T.handle, indptr: T.handle, indices: T.handle) -> None:
+    n = T.var("int32")
+    m = T.var("int32")
+    k = T.var("int32")
+    nnz = T.var("int32")
+    I = T.dense_fixed("I", n, "int32")
+    J = T.sparse_variable("J", (m, nnz), (indptr, indices), "int32")
+    K = T.dense_fixed("K", k, "int32")
+    A = T.match_sparse_buffer(a, (I, J), "float32")
+    B = T.match_sparse_buffer(b, (T.to_dense(J), K), "float32")
+    C = T.match_sparse_buffer(c, (I, K), "float32")
+    with T.iter((T.cord(I), T.cord(J), T.cord(K)), "SRS", "csrmm") as [vi, vj, vk]:
```

Comment:
Looks like the "special statement" for `T.iter` hasn't been pushed yet?

Reply:
yep. It should be binded with SparseBlock.

Reply:
Okay. Let me do this part.